.ado/jobs/cli-init.yml

```diff
@@ -99,6 +99,18 @@ parameters:
             platform: x86
             projectType: app
             useNuGet: true
+          - Name: X64DebugCppLowResource
+            language: cpp
+            configuration: Debug
+            platform: x64
+            projectType: app
+            lowResource: true
+          - Name: X64ReleaseCppLowResource
```

Comment:
Should remove release from PR, since it takes almost as long as experimental NuGet, and a single job should be enough to provide value. Especially if release could be more intensive and thus flakey.